README.md

```diff
@@ -88,52 +88,110 @@ curl -L https://aka.ms/install-dmr-client-linux | bash
 iwr https://aka.ms/install-dmr-client-windows -UseBasicParsing | iex
 ```
 
+#### Update / Uninstall
+
+To update the client you must uninstall the current version first:
+
+```
+dotnet tool uninstall -g microsoft.iot.modelsrepository.commandline
+```
+
 ### Import a Model to the `dtmi/` folder
 
 If you have your model already stored in json files, you can use the `dmr-client import` command to add those to the `dtmi/` folder with the right file name.
 
 ```bash
 # from the local repo root folder
-dmr-client import --model-file "MyThermostat.json"
+
+> dmr-client import --model-file "MyThermostat.json"
 ```
 
->Note: You can use the `--local-repo` argument to specify the local repo root folder
+> This command will rename and locate the file in the appropriate folder
 
 ### Validate Models
 
 You can validate your models with the `dmr-client validate` command.
 
 ```bash
-dmr-client validate --model-file ./my/model/file.json
+# Validate a model file using the DTDL parser
+
+> dmr-client validate --model-file ./my/model/file.json
 ```
 
 >Note: The validation uses the latest DTDL parser version to ensure all the interfaces are compatible with the DTDL language spec
 
 To validate external dependencies, those must exist in the local repo. To validate those you can specify a `local` or `remote` folder to validate against.
 
 ```bash
-# from the repo root folder
-dmr-client validate --model-file ./my/model/file.json --repo .
+# Validate a model file using the DTDL parser checking dependencies with the current folder as a local repo
+
+> dmr-client validate --model-file ./my/model/file.json --repo .
 ```
 
 #### Strict validation
 
 The Device Model Repo includes additional [requirements](pr-reqs.md), these can be validated with the `strict` flag.
 
 ```bash
-dmr-client validate --model-file ./my/model/file.json --repo . --strict true
+# Validate a model file using the DTDL parser checking dependencies with the current folder as a local repo in strict mode
+
+> dmr-client validate --model-file ./my/model/file.json --repo . --strict true
```

Comment:
you don't need to explicitly pass `--strict true` , `--strict` should be equivalent.